app/javascript/controllers/app_inspector_controller.js

```diff
@@ -0,0 +1,9 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["form"];
+
+  submitForm(event) {
+    this.formTarget.requestSubmit()
```

Comment:
Sadly I think this is what we're looking to do, which is kinda confusing. and we're talking about it now.

```javascript
 fetch(href, {
      headers: {
        Accept: "text/vnd.turbo-stream.html",
      },
    })
      .then(r => r.text())
      .then(html => Turbo.renderStreamMessage(html))
```